package.json

```diff
@@ -42,6 +42,8 @@
     "postcss-loader": "^1.3.3",
     "sass-loader": "^6.0.3",
     "style-loader": "^0.16.0",
+    "turf-distance": "^3.0.12",
```

Comment:
I think the package.json needs to drop mapbox.js and add mapbox-gl...we recently switched to yarn instead of npm for package management, so the simplest way to achieve that is, after installing yarn, `yarn add mapbox-gl` and `yarn remove mapbox.js`...besides the package.json, it will make a bunch of changes to the yarn.lock file.